scripts/userBio.js

```diff
@@ -0,0 +1,41 @@
+// randomizers helper functions
```

Comment:
super excited that you were able to use modules in this! 🙌 Everything will look so much cleaner! 💪